cockroachcloud/use-managed-service-backups.md

```diff
@@ -125,6 +124,20 @@ Additional ways to restore data:
 - [Back up a self-hosted CockroachDB cluster and restore into a {{ site.data.products.db }} cluster](#back-up-a-self-hosted-cockroachdb-cluster-and-restore-into-a-cockroachdb-cloud-cluster)
 - [Back up and restore data manually](take-and-restore-customer-owned-backups.html)
 
+### Restore a cluster
+
+To restore a cluster:
+
+1. Find the cluster backup on the **Backups** page.
```

Comment:
I'm not a markdown expert, would you confirm these items should all say 1? :)

Reply:
I'm not sure why they are all '1.' but that's how our markdown renders numbered lists, you can view in the Preview! :)